### DISCUSSION TITLE
Volume equalization?

### DISCUSSION THREAD

**Discussion by Huecuva** [MusicPlayerDaemon/MPD#2127] [Q&A]
I have a small server set up with several mpd streams. The streams themselves are all working fine, but the problem is that the volume varies based on the bitrate of the MP3 being played. Some tracks are louder than others and some can barely be heard at all. I know the MP3s should all be at least 192kbps themselves, most being 320kbps. I have mpd configured to stream them at 128kbps so it doesn't need to buffer as often, as that was an issue I was originally having. However, the volume issue persists. Is there a way to filter mpd through some kind of equalizer to adjust the volume so every track is about the same volume?

**Comment by Huecuva** [comment]
Nothing?

**Comment by jcorporation** [comment]
You can configure mpd to output to alsa, pulse, etc.

I think the moode audio project uses Camilla dsp with mpd.

I do not know if this helps.

Replay gain is responsible for adjusting the volume.

**Reply by Huecuva** [reply]
Thanks. I'll have a look into this.
